bin/ghe-restore

```diff
@@ -108,39 +108,27 @@ ghe_remote_version_required "$GHE_HOSTNAME"
 
 # Figure out if this instance has been configured or is entirely new.
 instance_configured=false
-if ghe-ssh "$GHE_HOSTNAME" -- \
-    "[ -f '$GHE_REMOTE_DATA_DIR/enterprise/dna.json' -o \
-       -f '$GHE_REMOTE_ROOT_DIR/etc/github/configured' ]"; then
-    instance_configured=true
-elif [ "$GHE_VERSION_MAJOR" -ge 2 ]; then
-    restore_settings=true
+if ghe-ssh "$GHE_HOSTNAME" -- "[ -f '$GHE_REMOTE_ROOT_DIR/etc/github/configured' ]"; then
+  instance_configured=true
+else
+  restore_settings=true
 fi
 
 # Figure out if we're restoring into cluster
-cluster=false
+CLUSTER=false
 if ghe-ssh "$GHE_HOSTNAME" -- \
   "[ -f '$GHE_REMOTE_ROOT_DIR/etc/github/cluster' ]"; then
-  cluster=true
+  CLUSTER=true
 fi
+export CLUSTER
 
 # Restoring a cluster backup to a standalone appliance is not supported
-if ! $cluster && [ "$GHE_BACKUP_STRATEGY" = "cluster" ]; then
+if ! $CLUSTER && [ "$GHE_BACKUP_STRATEGY" = "cluster" ]; then
```

Comment:
This currently blocks the restoration of a backup taken from a HA pair to a standalone appliance.

Reply:
Should be fixed now, and with a test too.